docs/evaluation.md

```diff
@@ -309,15 +308,15 @@ field and an optional `reference` field, like below:
 If your flow requires auth, you may specify it using the `--auth` argument:
 
 ```posix-terminal
-genkit eval:flow qaFlow --input testInputs.json --auth "{\"email_verified\": true}"
+genkit eval:flow qaFlow --input testInputs.json --auth '{"auth":{"email_verified":true}}'
```

Comment:
--context?

Reply:
Fixed in https://github.com/firebase/genkit/pull/1822